src/components/patch/PatchContainer.tsx

```diff
@@ -67,6 +67,14 @@ export class PatchContainer extends React.Component<Props, State> {
 
   private loadPatches = () => {
     if (this.state.hasMore && !this.state.isSearching) {
+      let username = this.props.username;
+      const urlParams = window.location.href.split("?");
```

Comment:
I would use URLSearchParams to parse this: https://davidwalsh.name/query-string-javascript